templates/collectd.conf.erb

```diff
@@ -0,0 +1,102 @@
+# Generated by Puppet
+# Install:
+#  You can usually install with apt-get or yum
+
+# Documentation:
+#   https://collectd.org/documentation.shtml
+
+# System modifications:
+#   None
+
+# Config file modifications:
+#   If you want to use a custom source name, change
+#   the "Hostname" paramter and set
+#
+#     FQDNLookup false
+#
+#   Make sure TypesDB points to a valid file:
+#
+#     which %%%COLLECTDSHARE%%%/types.db
+#
+#   Create the directory managed_config and put
+#   extra config files there:
+#
+#     mkdir /etc/collectd/managed_config
+#
+
+TypesDB "/usr/share/collectd/types.db"
+
+#   If you use the Hostname parameter, you
+#   must quote ("...") the second argument
+### Hostname "<YOUR_HOSTNAME_HERE>"
+
+<% if @fqdnlookup %>
+FQDNLookup true
+<% else %>
+Hostname "<%= @hostname %>"
+FQDNLookup false
+<% end %>
+Interval <%= @interval %>
+Timeout <%= @timeout %>
+ReadThreads <%= @read_threads %>
+WriteQueueLimitHigh <%= @write_queue_limit_high %>
+WriteQueueLimitLow  <%= @write_queue_limit_low %>
+CollectInternalStats <%= @collect_internal_stats %>
+
+LoadPlugin logfile
+
+<Plugin logfile>
+  LogLevel "info"
```

Comment:
in chef this is set to err instead of info.  that prevents flooding the log with information messages (like on centos and amazon where permission info messages and other unwanted things get added) while collecting bootstrap and error messages.